.idea/watcherTasks.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should probably be added to .gitignore so it doesn't get included in source control. It's from the IDE.

Reply:
Here is the gitignore I use:

*.terraform
.terraform.lock.hcl
*.tfstate
*node_modules
*.log
.serverless
.webpack
.DS_Store
.idea/
.vs/
yarn-error.log